client/app/cloud/project/compute/infrastructure/iac/view/translations/Messages_fr_FR.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<translations>
+    <translation id="cpciiac_view_title">Choisissez votre infrastructure à déployer</translation>
```

Comment:
Misspellings

Reply:
Please could you tell what you had in mind? Thanks

Reply:
It meant: the text should be supplied to `Content` so they can fix it

Reply:
> (...) so they can fix it

According to your previous comment, could you please be more precise? 
Where did you spot a misspelled sentence?